### DISCUSSION TITLE
Transferring the repo & maintainership

### DISCUSSION THREAD

**Discussion by crimson-knight** [amberframework/liquid.cr#61] [Ideas]
Hey there fellow Crystalists!

The liquid templating language is a big part of the ruby world and I want to continue to the development of this shard to include liquid in the crystal world as well! It's already a part of Amber (which I'm one of the primary maintainers of these days).

I proposed transferring this repo over to me to make facilitating maintenance of it easier and possibly include it in the Amber Framework repo in the future.

Does anyone object to this? Or alternative suggestions?

**Comment by HCLarsen** [comment]
I'm fully in favour of seeing the repo transferred to someone who will keep it active and maintained.

As you point out, Liquid is popular in the Ruby community, so it will likely be important to Crystalists as well, and it would be a big shame if someone had to start over developing a new Shard, instead of building upon this impressive work.

**Comment by hugopl** [comment]
IMO if it should move to somewhere, better be to amber project where more eyes can review the merge requests and design decisions.

**Reply by crimson-knight** [reply]
I think it would have to be two transfer steps to get it into Amber. I already have the i18n.cr shard to re-work into Amber and then I'll add it back to the main framework/org repo. It'd be nice to see everything used under one umbrella

**Reply by HCLarsen** [reply]
The only caveat I'll give is that because Liquid.cr is important for much more than just Amber, it's very important to keep it in a its own repo, and not mono-repo it with Amber.

**Reply by crimson-knight** [reply]
Yeah agreed, and I'd like to just move it into the org but not mono-repo it anyway. Amber is already built that way with it's other dependencies so it makes sense to keep moving in that direction

**Reply by hugopl** [reply]
BTW multiple persons already have the power to merge things here. Last PR was merged by me after it get two approvals, none of them was from the original shard author. I’m afraid of this being the start of a split that this repo already had in the past and was “fixed” by https://github.com/TechMagister/liquid.cr/pull/53

**Reply by hugopl** [reply]
> The only caveat I'll give is that because Liquid.cr is important for much more than just Amber, it's very important to keep it in a its own repo, and not mono-repo it with Amber.

Yeah, I was talking about a move to the amber framework github organization, not to merge this repo into Amber framework git repo :-)

**Reply by crimson-knight** [reply]
Yeah Amber uses another fork of this shard in it's dependencies now, I plan to remove it and focus on one whether it gets moved or not.

A lot of people find the amber framework org on here so it would help with focus and adoption for the shard to be that visible.

**Reply by hugopl** [reply]
IIRC all changes in amber fork got included in this repo when #56 got merged. Same for other forks.

**Reply by crimson-knight** [reply]
Amber is actually using another fork 
```yml
liquid:
    github: dare892/liquid.cr
    branch: chore/bump-version-crystal1.0.0
```

Amber originally used this shard, then made their own fork https://github.com/amberframework/i18n.cr----old-fork -- which says it's a fork from my repo, but that's because TechMagister transferred the i18n.cr shard to me over a year ago. I had made some tweaks on it with Imdrasil who maintains Jennifer.

I renamed it from the original i18n.cr because it was preventing me from transferring the existing i18n.cr shard into the Amber framework. I don't see much of a reason to keep it, so I'll probably just delete it soon. The stats were completely flat for anyone using it and it's been archived for a while.

**Comment by ArFnds** [comment]
Hello
As mentioned https://github.com/TechMagister/liquid.cr/discussions/63, I'll need help from a @amberframework owner
